racket-poppler/render-tex.rkt

```diff
@@ -44,15 +44,15 @@
   (set! defaultdir
         (or defaultdir
             (case (system-type)
-              [(macosx) "/Library/TeX/texbin"]
+              [(macosx) "/usr/texbin"]
```

Comment:
submitted new pull request with a way to do this.

On Tue, Feb 23, 2016 at 8:41 PM Jens Axel Søgaard <notifications@github.com>
wrote:

> In racket-poppler/render-tex.rkt
> <https://github.com/soegaard/racket-poppler/pull/8#discussion_r53846373>:
>
> > @@ -44,15 +44,15 @@
> >    (set! defaultdir
> >          (or defaultdir
> >              (case (system-type)
> > -              [(macosx) "/Library/TeX/texbin"]
> > +              [(macosx) "/usr/texbin"]
>
> Hmm - that means we need a way to figure out whether we are on El Capitan
> or earlier.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/soegaard/racket-poppler/pull/8/files#r53846373>.
>